test/e2e/nodes_test.go

```diff
@@ -39,7 +39,6 @@ var _ = Describe("Nodes", func() {
 				obtainedInterfaces := interfacesName(interfaces)
 				Expect(obtainedInterfaces).To(SatisfyAll(
 					ContainElement("eth0"),
-					ContainElement("eth1"),
```

Comment:
there is a major problem with this. everyone has eth0, even a simple pod. hopefully will be fixed once we are able to create interfaces (or request on local cluster)